vaas/vaas/metrics/handler.py

```diff
@@ -0,0 +1,28 @@
+from datetime import timedelta
+from django.conf import settings
+from vaas.metrics.prometheus import PrometheusMetrics
+from vaas.metrics.statsd import StatsdMetrics
+
+
+class MetricsHandler:
+    def __init__(self) -> None:
+        self.prometheus = PrometheusMetrics()
+        self.statsd = StatsdMetrics()
```

Comment:
Does it make sense to create instances for metric storage that is disabled?

Reply:
I would rather pass dependencies via parameters

Reply:
True. There will be more clear where this object are created.